gradle.properties

```diff
@@ -1,20 +1,22 @@
 # Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
+org.gradle.parallel=true
 
 # Fabric Properties
-# check these on https://modmuss50.me/fabric.html
+# Check these on https://fabricmc.net/develop/
 minecraft_version=1.20.1
-yarn_mappings=1.20.1+build.2
-loader_version=0.14.21
+yarn_mappings=1.20.1+build.10
+loader_version=0.14.25
+
 # Mod Properties
-mod_version=1.0+1.20.1
```

Comment:
Please revert this; it's very useful for archiving multiple versions and it's fully supported by Fabric's mod schema. It's even used by [Fabric API](https://github.com/FabricMC/fabric/)

Reply:
The Minecraft version is automatically appended in `build.gradle`, making the end result practically identical to that of specifying the Minecraft version in the `mod_version` in `gradle.properties` manually, but with one fewer extra step.

https://github.com/Luligabi1/EnhancedWorkbenches/blob/7c9f07989413e29a894fc2f77bf60a614294c800/build.gradle#L6

I can still revert this change though, if you'd like.

Reply:
ahh sorry, didn't see that. It *is* superior, but I'd probably forget this one mod has it done this way, ending up with stuff like `1.0+1.21+1.21` 😅 it's better to revert it

Reply:
Fair enough! I'll revert it then.